src/WhatToCook.Application/Services/ApllicationService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace WhatToCook.Application.Services;
+
+public static class ApllicationServicesDependencies
```

Comment:
Typo and filename is different than the class name. Please update filename to be the same as class name

Reply:
done, will be in the next commit